### PR TITLE
fix(schema.rb): update schema

### PR DESCRIPTION
**Pull request by LeeRenJie**
depends on #71

### CODE REVIEW

File: alpha-blog/db/schema.rb [hunk 1]
```diff
@@ -20,8 +20,8 @@
   end
 
   create_table "articles", force: :cascade do |t|
-    t.string "title"
```
Comment (tlkiong) [inline alpha-blog/db/schema.rb:23]: Where did all these changes come from?

Comment (LeeRenJie) [reply alpha-blog/db/schema.rb:23]: I think its from this [PR](https://github.com/LeeRenJie/Learn-Ruby/pull/15/files) but no idea why the update in migration files were not reflected on the schema. Pretty sure I ran `rails db:migrate` a couple of times after that. But just now it appeared after I ran `rails db:migrate` to run and see the app again.

**Review by tlkiong — CHANGES_REQUESTED** [review]
how did the schema magically get updated here?

**Comment by LeeRenJie** [comment]
> how did the schema magically get updated here?

No idea, I thought that the only reason for this to happen is from the migration file. I looked back at all the PR history for the migration file and saw the one I linked above is the only one that would make this happen. I was trying to refer to alpha blog just now to code the message-me myself before seeing the solution, ran `rails db:migrate` then realize there are changes in the `schema.rb`

**Review by tlkiong — APPROVED** [review]

**Pull request merged**